src/pages/Home.jsx

```diff
@@ -1,91 +1,141 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 import faker from 'faker'
 import React from 'react'
+import { useState } from 'react'
+import { useQueryClient, useQuery } from 'react-query'
+import axios from 'axios'
+import classNames from 'classnames'
 
 function Home() {
+  const PAGE_LIMIT = 5
+  const [offset, setOffset] = useState(0)
+  const [selectedTag, setSelectedTag] = useState(null)
```

Comment:
Maybe don't provide a default value here to make it `undefined`

Reply:
That's a possibility I'll check the behavior

Reply:
Seems you are right
https://github.com/axios/axios/blob/e9965bfafc82d8b42765705061b9ebe2d5532493/lib/helpers/buildURL.js#L38